src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.task.TaskList;
+
+/**
+ * A chatbot based on Project Duke

```

Comment:
Need punctuation behind each line of header comments, including parameters as well.